jdaviz/configs/imviz/tests/test_catalogs.py

```diff
@@ -157,7 +158,9 @@ def test_plugin_image_with_result(self, imviz_helper, tmp_path):
         assert imviz_helper.viewers['imviz-0']._obj.state.y_min == -0.5
         assert imviz_helper.viewers['imviz-0']._obj.state.y_max == 1488.5
 
-        catalogs_plugin.vue_zoom_in()
+        # set 'padding' to reproduce original hard-coded 50 pixel window
+        # so test results don't change
+        catalogs_plugin.zoom_to_selected(padding=50 / 2048)
```

Comment:
But why the `/ 2048`?

Reply:
thats the size of the image. the default zoom box size now depends on the size of the image rather than being a fixed 50 pixels. I wanted the test results to match the previous default of 50 so i set the percentage padding accordingly